src/MaterialReactTable.tsx

```diff
@@ -945,6 +945,7 @@ export type MaterialReactTableProps<TData extends Record<string, any> = {}> =
       table: MRT_TableInstance<TData>;
       values: Record<LiteralUnion<string & DeepKeys<TData>>, any>;
     }) => Promise<void> | void;
+    onEditingRowCancel?: OnChangeFn<MRT_Row<TData> | null>;
```

Comment:
This won't be the `onChange<>` type, will need to change it probably

Reply:
Yeah ....`onChange<>` maybe not best type for describing what is happening.  I thought returning the row might be some value possibly for messaging/logging purposes.